pytensor/link/c/cmodule.py

```diff
@@ -271,6 +272,14 @@ def _get_ext_suffix():
     return dist_suffix
 
 
+@cache
```

Comment:
Maybe add a comment of why we're caching this as it doesn't really make sense otherwise.

Reply:
Great point, done!